src/field_spec.h

```diff
@@ -108,19 +111,24 @@ typedef struct FieldSpec {
       // Flags for tag options
       TagFieldFlags tagFlags : 16;
       char tagSep;
+      struct TagIndex *tagIndex;
     } tagOpts;
     struct {
       // Vector similarity index parameters.
       VecSimParams vecSimParams;
       // expected size of vector blob.
       size_t expBlobSize;
+      VecSimIndex *vecSimIndex;
     } vectorOpts;
     struct {
       // Geometry index parameters
       GEOMETRY_COORDS geometryCoords;
+      GeometryIndex *geometryIndex;
     } geometryOpts;
   };
 
+  // TODO: Move into union above when we stop supporting multi-type fields
```

Comment:
@GuyAv46 You mean fields with option `FieldSpec_Dynamic`?

Reply:
Yes. Referring here to the text-only data and the numeric pointer